mirrormanager2/api.py

```diff
@@ -95,3 +95,63 @@ def api_mirroradmins():
     jsonout = flask.jsonify(output)
     jsonout.status_code = 200
     return jsonout
+
+
+@APP.route('/api/repositories/')
+@APP.route('/api/repositories')
```

Comment:
It's not required per say, it's just the way to ask flask to support both URL, with and without the trailing slash. After it's a matter of personal preferences if you use it or not :)